resources/fetchers/process_fetcher.go

```diff
@@ -2,18 +2,27 @@ package fetchers
 
 import (
 	"context"
-
+	"errors"
+	"fmt"
+	"github.com/elastic/beats/v7/libbeat/logp"
 	"github.com/elastic/beats/v7/x-pack/osquerybeat/ext/osquery-extension/pkg/proc"
+	"k8s.io/apimachinery/pkg/util/json"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"os"
+	"path/filepath"
+	"regexp"
 )
 
 const (
-	ProcessType = "process"
+	ProcessType        = "process"
+	CMDArgumentMatcher = "\\b%s=(\\S+)"
 )
 
 type ProcessResource struct {
-	PID  string        `json:"pid"`
-	Cmd  string        `json:"command"`
-	Stat proc.ProcStat `json:"stat"`
+	PID    string                 `json:"pid"`
+	Cmd    string                 `json:"command"`
+	Stat   proc.ProcStat          `json:"stat"`
+	Config map[string]interface{} `json:"config"`
```

Comment:
Wondering about the name of the field. 

Will we ever extend it?

Reply:
```suggestion
	Config common.MapStr `json:"config"`
```

Reply:
🤔